etc/docker/tensorflow/1.15.3-gpu-py3/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM tensorflow/tensorflow:1.15.2-gpu-py3
+
+RUN apt install -y curl
```

Comment:
We should probably add a newline here otherwise diff tools have a hard time with things.